ngStorage.js

```diff
@@ -16,7 +19,7 @@
      * @name ngStorage
      */
 
-    angular.module('ngStorage', [])
+    exports = angular.module('ngStorage', [])
```

Comment:
This won't work, you just re-assign the local variable, but you do not "export" anything.
Notice that in CommonJS you have to put `module.exports = something` to re-assign the property of the `module` object, not the local variable `exports` that initially refers to the same object as `module.export`.

Initially:
```
module.exports ----> { /* the exported
exports ----------->      object */ }
```

After `exports.someObject = { /* some other object */ };`
```
module.exports ----> { /* the exported
exports ----------->      object */ someObject: { /* some other object */ } }
```

After `exports = { /* some other object */ };`
```
module.exports ----> { /* the exported object */ }
exports -----------> { /* some other object */ }
```